controls/raddocking/features/document-host.md

```diff
@@ -123,6 +123,28 @@ __Figure 2:__ __DocumentHost__ with visible __HorizontalScrollBar__ and drop dow
 
 ![](images/raddocking-features-document-host-6.png)
 
+## Removing Default Padding
+
+To remove the extra space around the panes in the pane groups and the `DocumentHost`, you need to create an appropriate style for the `RadPaneGroup` element and override the default `Padding`.
+
+> For the Fluent, Green, Material, Office2016 and Office2016Touch themes, the `DocumentHost` element's padding should be updated through a style trigger.
+
+#### __XAML__
+
+{{region xaml-raddocking-features-document-host_4}}
+	<!-- If you're using the NoXaml binaries, you need to base the custom style on the default one for the control, like so:
+	<Style TargetType="telerik:RadPaneGroup" BasedOn="{StaticResource RadPaneGroupStyle}"> -->

```

Comment:
Show where this Style is defined. I will guess it is an implicit one, so you can wrap it in a Window.Resources or UserControl.Resources, or Application.Resources tag.

The same feedback (+ the previous comment) goes for the KB article as well.